src/Sylius/Component/Core/Calculator/ProductVariantPriceCalculator.php

```diff
@@ -50,29 +43,30 @@ public function calculateOriginal(ProductVariantInterface $productVariant, array
         $channelPricing = $productVariant->getChannelPricingForChannel($context['channel']);
 
         if (null === $channelPricing) {
-            throw new MissingChannelConfigurationException(sprintf(
-                'Channel %s has no price defined for product variant %s',
-                $context['channel']->getName(),
-                $productVariant->getName()
-            ));
+            throw new MissingChannelConfigurationException($this->getMissingChannelProductVariantPriceMessage($context['channel'], $productVariant));
         }
 
         if (null === $channelPricing->getOriginalPrice()) {
```

Comment:
This can also return early.
```suggestion
        if (null !== $channelPricing->getOriginalPrice()) {
            return $channelPricing->getOriginalPrice();
        }
```